.github/workflows/main.yml

```
@@ -9,6 +9,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: tab character check
+        run: if grep -P '\t' -r ices/* fixed/*; then exit 1; fi
```

sounds good, just a small change to make it easier to know where to look from the CI logs (adds line numbers)

```suggestion
        run: if grep -P '\t' -rn --color ices fixed; then exit 1; fi
```